ui/lib/pki/addon/components/pki-generate-csr.hbs

```diff
@@ -32,9 +32,10 @@
                 <LinkTo @route="keys.key.details" @models={{array @model.backend @model.keyId}}>
                   {{@model.keyId}}
                 </LinkTo>
+              {{else if (value)}}
+                <span>{{"value"}}</span>
               {{else}}
-                {{! this block only ever renders privateKey and privateKeyType }}
```

Comment:
I think we should keep this comment since there's a lot of nuance to PKI. Maybe below the `{{else if}}` we can say something like 
```
{{! it's unlikely but if a value is returned for privateKey and privateKeyType we want to display it, otherwise we show the "internal" badge below }}